internal/config/config.go

```diff
@@ -43,11 +47,20 @@ func Load(args []string) (Config, error) {
 	fs.StringVar(&cfg.scheme, "scheme", "https", "using http or https")
 	fs.StringVar(&cfg.host, "host", "0.0.0.0", "host to bind to")
 	fs.StringVar(&cfg.Seed, "data", "", "where to load data from (provided that the directory exists)")
+	fs.StringVar(&allowedCORSHeaders, "cors-headers", "", "comma separated list of headers to add to the CORS allowlist")
 	fs.UintVar(&cfg.port, "port", 4443, "port to bind to")
+
 	err := fs.Parse(args)
 	if err != nil {
 		return cfg, err
 	}
+
+	if len(allowedCORSHeaders) > 0 {
+		cfg.allowedCORSHeaders = strings.Split(allowedCORSHeaders, ",")
+	} else {
+		cfg.allowedCORSHeaders = []string{}
+	}
```

Comment:
Let's check if the string is not empty and remove the else block.

```suggestion
	if allowedCORSHeaders != "" {
		cfg.allowedCORSHeaders = strings.Split(allowedCORSHeaders, ",")
	}
```

Reply:
Ha! What was I thinking 🤦🏽